lib/active_admin/string_input_filters_patch.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ActiveAdmin
+  module Inputs
+    module Filters
+      module StringInputExt
+        def self.included(base)
+          base.class_eval do
+            @filters = %i[equals contains starts_with ends_with]
+          end
+        end
+      end
+    end
+  end
+end
+
+ActiveAdmin::Inputs::Filters::StringInput.send(:include, ActiveAdmin::Inputs::Filters::StringInputExt)
```

Comment:
please rearrange filters only on /cdrs page

something like this
```ruby
filter :src_prefix_in, filters: %i[equals contains starts_with ends_with]
```
for all string filters with a predicate dropdown